docs/data/data-grid/overlays/LoadingOverlaySkeleton.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import Box from '@mui/material/Box';
 import { useDemoData } from '@mui/x-data-grid-generator';
-import { DataGridPro } from '@mui/x-data-grid-pro';
+import { DataGrid } from '@mui/x-data-grid';
```

Comment:
Can you add another demo with a skeleton loading overlay and pinned columns just for visual regression testing?
It doesn't have to be used in the docs.